src/Sarif/Visitors/RebaseUriVisitor.cs

```diff
@@ -34,6 +34,11 @@ public RebaseUriVisitor(string uriBaseId, Uri baseUri, bool rebaseRelativeUris =
 
         public override ArtifactLocation VisitArtifactLocation(ArtifactLocation node)
         {
+            if (node.Uri == null)
+            {
+                return node;
+            }
+
             if (_currentRun == null || _currentRun.OriginalUriBaseIds == null || !_currentRun.OriginalUriBaseIds.Values.Contains(node))
```

Comment:
>if [](start = 12, length = 2)

With the C# 6 null conditional operator, this simplifies to
```
if (_currentRun?.OriginalUriBaseIds?.Values.Contains(node) != true)
```
The explicit test is necessary because the return type of the operator is `bool?` (`Nullable<bool>`), and there's no implicit conversion from `bool?` to the `bool` that the `if` conditional requires.